tests/unit_tests/strictload_enum_test.py

```diff
@@ -54,7 +54,7 @@ def setUpClass(cls):
 
         # Save the model's state_dict checkpoint in Trainer format
         cls.trainer = Trainer("load_checkpoint_test", model_checkpoints_location='local')  # Saves in /checkpoints
-        cls.trainer.build_model(cls.original_torch_model, arch_params={'num_classes': 10})
+        cls.trainer.set_net(cls.original_torch_model)
```

Comment:
what is this?

Reply:
we nee to simulate checkpoint saving that occurs during training, without changing the models weights so we can compare them.
Beforehand, we could just pass the net to build_model, and call save_checkpoint. Now, we have to set the net manually before calling it.